prowler/providers/aws/services/organizations/organizations_service.py

```diff
@@ -0,0 +1,231 @@
+from prowler.providers.aws.aws_provider import generate_regional_clients
+
+
+################## Organizations
+class Organizations:
+    def __init__(self, audit_info):
+        self.service = "organizations"
+        self.session = audit_info.audit_session
+        self.audited_account = audit_info.audited_account
+        self.audit_resources = audit_info.audit_resources
+        global_client = generate_regional_clients(
+            self.service, audit_info, global_service=True
+        )
+        self.client = list(global_client.values())[0]
+        self.region = self.client.region
+        self.organizations = []
+        self.policies = []
+        self.delegated_administrators = []
+        self.__describe_organization__()
+
+    def __describe_organization__(self):
+        logger.info("Organizations - Describe Organization...")
+
+        try:
+            # Check if Organizations is in-use
+            try:
+                organization_desc = self.client.describe_organization()["Organization"]
+                organization_arn = organization_desc["Arn"]
+                organization_id = organization_desc["Id"]
+                organization_master_id = organization_desc["MasterAccountId"]
+                organization_available_policy_types = organization_desc[
+                    "AvailablePolicyTypes"
+                ]
+                # Fetch policies for organization:
+                organization_policies = self.__list_policies__(
+                    organization_available_policy_types
+                )
+                # Fetch delegated administrators for organization:
+                organization_d_administrator = self.__list_delegated_administrators__()
+            except ClientError as e:
+                if e.response["Error"]["Code"] == "AWSOrganizationsNotInUseException":
+                    self.organizations.append(
+                        Organization(
+                            arn="", id="", status="NOT_AVAILABLE", master_id=""
+                        )
+                    )
+            else:
+                if not self.audit_resources or (
+                    is_resource_filtered(organization_arn, self.audit_resources)
+                ):
+                    self.organizations.append(
+                        Organization(
+                            arn=organization_arn,
+                            id=organization_id,
+                            status="ACTIVE",
+                            master_id=organization_master_id,
+                            policies=organization_policies,
+                            delegated_administrators=organization_d_administrator,
+                        )
+                    )
+                else:
+                    # is filtered
+                    self.organizations.append(
+                        Organization(
+                            arn="", id="", status="NOT_AVAILABLE", master_id=""
+                        )
+                    )
```

Comment:
Why this here? We are not storing the resource if we don't want to audit it.

Reply:
Agree that this could be confusing, but this was on purpose: What we are defining here is that organization is not in use. `self.organizations` is a list, but it will always only one element (there is only a maximum one organization by account), what I'm doing here is adding a dummy organization in case the service is disabled. The reason why I'm using a list here is to be consistent with all the other checks, like here: https://github.com/prowler-cloud/prowler/blob/master/prowler/providers/aws/services/securityhub/securityhub_service.py#L88

Reply:
You're right, thanks for the explanation!